diff --git a/node_modules/ffjavascript/build/browser.esm.js b/node_modules/ffjavascript/build/browser.esm.js
index 3ceee4f..f7d066e 100644
--- a/node_modules/ffjavascript/build/browser.esm.js
+++ b/node_modules/ffjavascript/build/browser.esm.js
@@ -18028,6 +18028,8 @@ class ModuleBuilder {
 globalThis.curve_bn128 = null;
 
 async function buildBn128(singleThread, plugins) {
+    singleThread = true;
+    console.log('browser.esm single thread true')
     if ((!singleThread) && (globalThis.curve_bn128)) return globalThis.curve_bn128;
 
     const moduleBuilder = new ModuleBuilder();
@@ -18171,6 +18173,8 @@ async function getCurveFromQ(q, singleThread, plugins) {
 
 async function getCurveFromName(name, singleThread, plugins) {
     let curve;
+    singleThread = true;
+    console.log('browser.esm single thread true')
     const normName = normalizeName(name);
     if (["BN128", "BN254", "ALTBN128"].indexOf(normName) >= 0) {
         curve = await buildBn128(singleThread, plugins);
diff --git a/node_modules/ffjavascript/build/main.cjs b/node_modules/ffjavascript/build/main.cjs
index a8bd342..03279e0 100644
--- a/node_modules/ffjavascript/build/main.cjs
+++ b/node_modules/ffjavascript/build/main.cjs
@@ -5839,6 +5839,7 @@ async function buildEngine(params) {
 globalThis.curve_bn128 = null;
 
 async function buildBn128(singleThread, plugins) {
+    singleThread = true;
     if ((!singleThread) && (globalThis.curve_bn128)) return globalThis.curve_bn128;
 
     const moduleBuilder = new wasmbuilder.ModuleBuilder();
@@ -5982,6 +5983,7 @@ async function getCurveFromQ(q, singleThread, plugins) {
 
 async function getCurveFromName(name, singleThread, plugins) {
     let curve;
+    singleThread = true;
     const normName = normalizeName(name);
     if (["BN128", "BN254", "ALTBN128"].indexOf(normName) >= 0) {
         curve = await buildBn128(singleThread, plugins);
diff --git a/node_modules/ffjavascript/main.js b/node_modules/ffjavascript/main.js
index 1981557..1ffec1d 100644
--- a/node_modules/ffjavascript/main.js
+++ b/node_modules/ffjavascript/main.js
@@ -2,6 +2,8 @@
 import * as _Scalar  from "./src/scalar.js";
 export const Scalar=_Scalar;
 
+console.log('ffjavascript main.js import')
+
 export {default as PolField} from "./src/polfield.js";
 export {default as F1Field} from "./src/f1field.js";
 export {default as F2Field} from "./src/f2field.js";
diff --git a/node_modules/ffjavascript/src/bn128.js b/node_modules/ffjavascript/src/bn128.js
index 7a421c1..c457e34 100644
--- a/node_modules/ffjavascript/src/bn128.js
+++ b/node_modules/ffjavascript/src/bn128.js
@@ -6,6 +6,7 @@ import { ModuleBuilder } from "wasmbuilder";
 globalThis.curve_bn128 = null;
 
 export default async function buildBn128(singleThread, plugins) {
+    singleThread = true;
     if ((!singleThread) && (globalThis.curve_bn128)) return globalThis.curve_bn128;
 
     const moduleBuilder = new ModuleBuilder();
